olive/systems/docker/docker_system.py

```diff
@@ -358,16 +369,25 @@ def _run_container(
 
         return Path(output_local_path) / output_name
 
-    def _create_data_mounts_for_pass(self, container_root_path: Path, the_pass: "Pass"):
+    def _create_data_mounts_for_pass(
+        self, container_root_path: Path, full_pass_config: "FullPassConfig"
+    ) -> Tuple[Dict[str, str], List[str]]:
         mounts = {}
         mount_strs = []
-        config_dict = the_pass.config.dict()
-        for param, _, category in the_pass.path_params:
-            param_val = config_dict.get(param)
-            if category == ParamCategory.DATA and param_val:
-                mount = str(container_root_path / param)
-                mounts[param] = mount
-                mount_strs.append(f"{param_val}:{mount}")
+        if not full_pass_config.config:
+            return mounts, mount_strs
+
+        olive_config = OlivePackageConfig.load_default_config()
+        pass_cls: Type[Pass] = olive_config.import_pass_module(full_pass_config.type)
+
+        config_dict = full_pass_config.to_json().get("config") or {}
+        for param, param_config in pass_cls.default_config(full_pass_config.accelerator).items():
```

Comment:
Little bit confusing here, can we use `full_pass_config.config` to get pass config?

Reply:
But we need a dictionary here not an object of type `BasePassConfig`.

Reply:
Is this the only way to get a dict config? I just realize one of our [test](https://aiinfra.visualstudio.com/PublicPackages/_build/results?buildId=730983&view=logs&j=04f3f8bd-f6ad-546f-a3e7-a621004c3b55&t=be946a69-691f-5f4e-fe5a-c2d62ee382d6&l=201) is failing due to this as well after this [pr](https://github.com/microsoft/Olive/pull/1567). Can you fix this?